src/generated/cloud/bigquery/datapolicies/v2/.sidekick.toml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/cloud/bigquery/datapolicies/v2'
+service-config = 'google/cloud/bigquery/datapolicies/v2/bigquerydatapolicy_v2.yaml'
+
+[codec]
```

Comment:
nit, and not that this matters: if we are breaking from one-version-to-rule-them-all, this would be `version = 0.0.1`, right?

Reply:
Hmmm... I think you are right. But let's keep it simple until we figure out more of the ins and outs of per-library versions.